### ISSUE TITLE
Compile widgetset fails with Vaadin 7.6.0.beta1

### ISSUE THREAD

**Issue by yoursnowyfriendleo** [johndevs/dragdroplayouts#42]
[echo] Compiling nz.co.ais.mtvic.gwt.Mywidgetset...
     [echo] ../Code/src/nz/co/ais/mtvic/gwt/
     [java] Compiling module nz.co.ais.mtvic.gwt.Mywidgetset
     [java]    Tracing compile failure path for type 'fi.jasoft.dragdroplayouts.client.ui.formlayout.DDFormLayoutConnector'
     [java]       [ERROR] Errors in 'jar:file:/C:/XYZ/Current/3rdParty/vaadin/libs/dragdroplayouts-1.1.3.jar!/fi/jasoft/dragdroplayouts/client/ui/formlayout/DDFormLayoutConnector.java'
     [java]          [ERROR] Line 40: Cannot cast from FormLayoutState to DDFormLayoutState
     [java]          [ERROR] Line 39: The return type is incompatible with FormLayoutConnector.getState()
     [java]    [ERROR] Aborting compile due to errors in some input files

Cheers, Simon

**Comment by johndevs** [comment]
Hi,

This was fixed in 6e053f824599c99727f30ad776389b87abc220f0 but has not been released yet.

You could try using the SNAPSHOT build in the following maven repository:
```
Repository:    http://mvn.jasoft.fi/maven2-snapshots/
Group id:      fi.jasoft
Artifact Id:   dragdroplayouts
Version:       1.2-SNAPSHOT
```

**Comment by yoursnowyfriendleo** [comment]
1.2-SNAPSHOT works with vaadin-7.6.0.beta2. Thank you very much.

Cheers, Simon
